README.md

```diff
@@ -1 +1,12 @@
-# trustly-android
\ No newline at end of file
+# Trustly Android SDK
+
+This repository contains all source code from our Android SDK, like interfaces and views that creates the integration with Trustly.
```

Comment:
"This repository contains the source code for the Trustly Android SDK, including interfaces and views that integrated with the web-based Trustly user interface module."